src/Components/Typography/BodyText/BodyText.js

```diff
@@ -17,7 +17,11 @@ class BodyText extends Component {
 
     const textStyle = type == 2 ? theme.bodyTextTwo : theme.bodyText;
     return (
-      <BaseText typographyStyles={textStyle} style={style} {...this.props}>
+      <BaseText
+        typographyStyles={textStyle}
+        style={style}
```

Comment:
Since there's no rename and you're already spreading props, there's no need to destructure `style` or `theme` unless you just want it to be evident that they are being used.

Reply:
I'd rather be explicit since these are things the library is passing and using, `{...this.props}` is to ensure anything else the user passes will be spread to the base `<Text />` component.